### ISSUE TITLE
Classification Beats Regression: Counting of Cells from Greyscale Microscopic Images based on Annotation-free Training Samples. (arXiv:2010.14782v1 [eess.IV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#108]
Classification Beats Regression: Counting of Cells from Greyscale Microscopic Images based on Annotation-free Training Samples. (arXiv:2010.14782v1 [eess.IV])<br>
https://ift.tt/3oybQ4W<br>
<p>Modern methods often formulate the counting of cells from microscopic images as a regression problem and more or less rely on expensive, manually annotated training images (e.g., dot annotations indicating the centroids of cells or segmentation masks identifying the contours of cells). This work proposes a supervised learning framework based on classification-oriented convolutional neural networks (CNNs) to count cells from greyscale microscopic images without using annotated training images. In this framework, we formulate the cell counting task as an image classification problem, where the cell counts are taken as class labels. This formulation has its limitation when some cell counts in the test stage do not appear in the training data. Moreover, the ordinal relation among cell counts is not utilized. To deal with these limitations, we propose a simple but effective data augmentation (DA) method to synthesize images for the unseen cell counts. We also introduce an ensemble method, which can not only moderate the influence of unseen cell counts but also utilize the ordinal information to improve the prediction accuracy. This framework outperforms many modern cell counting methods and won the data analysis competition (Case Study 1: Counting Cells From Microscopic Images https://ift.tt/3e6XsMn) of the 47th Annual Meeting of the Statistical Society of Canada (SSC). Our code is available at https://ift.tt/2JaWOBP.</p>
<br>
<br>
via cs.CV updates on arXiv.org http://arxiv.org/<br>
